From 797eadd9c80ca3b3f913ccde29f8a6015f9974f9 Mon Sep 17 00:00:00 2001
From: Alexey Kardashevskiy <aik@ozlabs.ru>
Date: Mon, 14 Jan 2019 11:41:38 +1100
Subject: [PATCH] powerpc/powernv/npu: Remove obsolete comment about
 TCE_KILL_INVAL_ALL

References: bsc#1065729
Patch-mainline: v5.1-rc1
Git-commit: 797eadd9c80ca3b3f913ccde29f8a6015f9974f9

TCE_KILL_INVAL_ALL has moved long ago but the comment was forgotted so
finish the move and remove the comment.

Fixes: 0bbcdb437da0c4a "powerpc/powernv/npu: TCE Kill helpers cleanup"
Signed-off-by: Alexey Kardashevskiy <aik@ozlabs.ru>
Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
Acked-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/platforms/powernv/npu-dma.c | 4 ----
 1 file changed, 4 deletions(-)

--- a/arch/powerpc/platforms/powernv/npu-dma.c
+++ b/arch/powerpc/platforms/powernv/npu-dma.c
@@ -46,10 +46,6 @@ static DEFINE_SPINLOCK(npu_context_lock)
  */
 #define ATSD_THRESHOLD (2*1024*1024)
 
-/*
- * Other types of TCE cache invalidation are not functional in the
- * hardware.
- */
 static struct pci_dev *get_pci_dev(struct device_node *dn)
 {
 	return PCI_DN(dn)->pcidev;
